deploy/rbac.yaml

```diff
@@ -45,7 +45,7 @@ metadata:
   name: default-account-marketplace-operator
 subjects:
 - kind: ServiceAccount
-  name: default
+  name: marketplace-operator-serviceaccount
   namespace: marketplace
```

Comment:
You would also need to change change the deployment spec to use this service account in `operator.yaml` file, I believe.
```
   spec:
      serviceAccountName: marketplace-operator-serviceaccount
```